src/demo/demoUtils.ts

```diff
@@ -1,14 +1,69 @@
+import { LocaleConfig, SupportedLanguages } from '~types/locales'
+import { StepConfig, StepTypes } from '~types/steps'
+import { ServerRegions, SdkOptions } from '~types/sdk'
+
+type StringifiedBoolean = 'true' | 'false'
+
+export type QueryParams = {
+  countryCode?: StringifiedBoolean
+  disableAnalytics?: StringifiedBoolean
+  forceCrossDevice?: StringifiedBoolean
+  hideOnfidoLogo?: StringifiedBoolean
+  language?: 'customTranslations' | SupportedLanguages
+  link_id?: string
+  liveness?: StringifiedBoolean
+  multiDocWithInvalidPresetCountry?: StringifiedBoolean
+  multiDocWithPresetCountry?: StringifiedBoolean
+  noCompleteStep?: StringifiedBoolean
+  oneDoc?: StepTypes
+  oneDocWithCountrySelection?: StringifiedBoolean
+  oneDocWithPresetCountry?: StringifiedBoolean
+  poa?: StringifiedBoolean
+  region?: string
+  shouldCloseOnOverlayClick?: StringifiedBoolean
+  showCobrand?: StringifiedBoolean
+  smsNumber?: StringifiedBoolean
+  snapshotInterva?: StringifiedBoolean
+  snapshotInterval?: StringifiedBoolean
+  uploadFallback?: StringifiedBoolean
+  useHistory?: StringifiedBoolean
+  useLiveDocumentCapture?: StringifiedBoolean
+  useMemoryHistory?: StringifiedBoolean
+  useModal?: StringifiedBoolean
+  useMultipleSelfieCapture?: StringifiedBoolean
+  useUploader?: StringifiedBoolean
+  useWebcam?: StringifiedBoolean
+}
+
+export type CheckData = {
+  applicantId: string | null
+  sdkFlowCompleted: boolean
+}
+
+export type ViewOptions = {
+  darkBackground: boolean
+  iframeWidth: string
+  iframeHeight: string
+  tearDown: boolean
+}
+
+const SAMPLE_LOCALE: LocaleConfig = {
+  phrases: { 'welcome.title': 'My custom title' },
+  mobilePhrases: {
+    'capture.driving_licence.back.instructions': 'Custom instructions',
+  },
+}
+
 export const queryParamToValueString = window.location.search
```

Comment:
Noticed you used `queryParamToValueString.<someting> as ...`, maybe better to type the object here?

Reply:
Originally I cast `queryParamToValueString` to be `QueryParams` at the end of the conversion, but today I found a better solution for it which declare the first param of `reduce` function (`acc` - the accumulated result) to be `QueryParams`